app/services/revert_rejected_by_default.rb

```diff
@@ -0,0 +1,44 @@
+class RevertRejectedByDefault
+  attr_reader :ids, :new_rbd_date
+
+  def initialize(ids:, new_rbd_date:)
+    @ids = ids
+    @new_rbd_date = new_rbd_date
+  end
+
+  def call
+    Audited.audit_class.as_user('RevertRejectedByDefault worker') do
+      ApplicationChoice
+        .where(application_form_id: ids)
+        .where(rejected_by_default: true)
+        .find_each do |application_choice|
+          statuses_for_form = application_choice.self_and_siblings.pluck(:status)
+
+          # do not continue if the application has an accepted offer
+          next if statuses_for_form.any? do |s|
+            ApplicationStateChange::ACCEPTED_STATES.include?(s.to_sym)
+          end
+
+          application_choice.update!(
+            reject_by_default_at: new_rbd_date,
+            status: :awaiting_provider_decision,
+            rejected_by_default: false,
+            rejected_at: nil,
+          )
+
+          application_choice.self_and_siblings.where(status: :offer).update_all(
+            decline_by_default_at: nil,
+            decline_by_default_days: nil,
+          )
+        end
+
+      ApplicationForm
+        .where(id: ids)
+        .find_each do |form|
+          form.chasers_sent
+            .where(chaser_type: :candidate_decision_request)
```

Comment:
nice one! are there any provider chasers we need to worry about?

Reply:
as discussed, because hols are <20wd and we'll be emailing providers about new RBDs anyway, we don't want to email them again with another chaser. So we'll leave the chasers alone.